Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download_PCAP_v2_README.md

```diff
@@ -0,0 +1,69 @@
+
+
+## Dependencies
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+* GenericPolling
+
+### Integrations
+* Endace
+
+### Scripts
+* AreValuesEqual
+* Print
+* AddEvidence
+* IsGreaterThan
+* Set
+
+### Commands
+* endace-get-archive-status
+* endace-create-search
+* endace-create-archive
+* endace-delete-archived-file
+* endace-download-pcap
+
+## Playbook Inputs
+---
+
+| **Name** | **Description** | **Default Value** | **Required** |
+| --- | --- | --- | --- |
+| start | UTC StartTime in ISO 8601 format as in 2020\-04\-08T15:46:30 |  | Optional |
+| end | UTC EndTime in ISO 8601 format  as in 2020\-04\-08T15:46:30 |  | Optional |
+| timeframe | Event timeframe to search. Select one of the values from  30seconds, 1minute, 5minutes, 10minutes, 30minutes, 1hour, 2hours, 5hours, 10hours, 12hours, 1day, 3days, 5days, 1week. Timeframe works as search for last n seconds if start and end time is not provided. For example, by specifying 1h as the timeframe, analyst can schedule a search for last 3600s. If both start and end time is provided, timeframe value is ignored. If either start or end time is provided along with timeframe, the respective start or end time is calculated accordingly. Initial value of timeframe is 1hour. |  | Optional |
+| ip | directionless ip address. For valid search either a IP or Src Host or a Dest Host value is required |  | Optional |
+| port | directionless port.  |  | Optional |
+| src_host_list | List of comma delimited Source IP addresses to search with a maximum of 10 IP addresses per search. For valid search either a Src Host or a Dest Host value is required. |  | Optional |
+| dest_host_list | List of comma delimited Destination IP addresses to search with a maximum of 10 IP addresses per search. For valid search either a Src Host or a Dest Host value is required. |  | Optional |
+| src_port_list | List of comma delimited Source Port addresses to search with a maximum of 10 Port addresses per search. |  | Optional |
+| dest_port_list | List of comma delimited Destination Port addresses to search with a maximum of 10 Port addresses per search. |  | Optional |
+| protocol | IANA defined IP Protocol Name or Number. For example \- either use TCP or tcp or 6 for tcp protocol |  | Optional |
+| archive_filename | Name of the archive file. For example, archive\_filename could be an event ID. To keep archive filename unique, value of epoch seconds at the time of execution of the command is appended to this filename argument. For example \- if the event id is someid, then archive\_filename is someid\-&lt;epochtime&gt;. |  | Required |
+| delete_archive | Delete archived file 0: Don't delete archived file. 1: Delete archived file. By default archived files on EndaceProbe won't be deleted |  | Required |

```

Comment:
@soumenchakraborty-endace 
As we discussed please consider improving the description or changing the values to something such as true/false for deletion of the pcap
In case you make the change please update the conditional tasks accordingly